description.md

```diff
@@ -34,6 +34,50 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Practical Sessions
 
+Practical sessions are represented by hands-on activities that learners perform in order to acquire practical experience.
+These activities are typically supervized by a teacher that proposes the practical exercices and evaluates the learner's performance.
+The purpose of the evaluation is mainly to provide constructive feedback, as opposed to grading the student.
+
+Instances of practical sessions are represented by laboratories, seminars or workshops.
+The format of a practical session may include:
+
+- theoretical and practical explanations offered by the teacher.
+- practical activities performed by the learners.
+- debates and discussions between students and teacher or between students themselves regarding specific topics.
+- feedback from the teacher or peers.
+
+The presence and length of each of these activities during a practical session is influenced by the preference of the teacher.
+As such, the educational materials used as support for the practical sessions should offer the flexibility of being tailored depending on the teacher's needs.
+At the same time, they should be usable in the absence of a teacher as standalone learning materials for self-taught individuals that want to learn by themselves.
+
+To meet these constraints, educational materials for practical sessions should abide to the following format.
+A practical sessions is comprised of a series of concepts that the learner must assimilate.
+A concept must contain:
+
+- a brief theoretical description (which may contain links to more broader explanations) by using reading and media materials.
+- one or more practical exemplifications of the described concept by using demos.
+- one or more tutorials where the learner makes the first steps in applying the described concept.
+- one or more quizzes regarding some aspects of what has been presented so far.
+- one or more drills where the learner applies the presented concept.
+
+There is no enforcement on the order in which the above content types should be present in the concept layout, because that depends on the nature of the presented concept.
```

Comment:
```suggestion
There is no enforcement on the order in which the above content types should appear in the concept layout because that depends on the nature of the presented concept.
```